docs/lib/project-setup/fragments/flutter/create-application/30_provisionBackend.md

```diff
@@ -1,6 +1,8 @@
 To start provisioning resources in the backend, change directories to your project directory and run `amplify init`:
 
 ```bash
+# in future, make sure you have Amplify CLI v4.28 and above for Flutter support
+# npm install -g @aws-amplify/cli@4.26.1-flutter-preview.0 for now
 amplify init
```

Comment:
latest version of CLI doesnt offer flutter support, so i am just inserting a version warning